lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationDataContract/AvroConfiguration.cs

```diff
@@ -37,7 +38,7 @@ public AvroConfiguration()
         }
 
         [DataMember]
-        public HashSet<ConfigurationEntry> Bindings { get; set; }
+        public List<ConfigurationEntry> Bindings { get; set; }
```

Comment:
I'm not exactly sure what the convention for REEF is, but since it was using HashSet before, I just used List directly instead of IList.